README.md

```diff
@@ -43,6 +43,10 @@ UCC is BSD-style licensed, as found in the [LICENSE](LICENSE) file.
    * UCC supports CUDA collectives. To compile with CUDA support, install
      [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.0 or above.
 
+* [HIP](https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html) (optional)
```

Comment:
looks mis-aligned  with [UCX] / [CUDA] above

Reply:
I'll fix that, probably again a space vs. tab issue